toolkit/tools/roast/roast.go

```diff
@@ -87,13 +94,18 @@ func main() {
 		logger.Log.Panicf("Failed loading image configuration. Error: %s", err)
 	}
 
+	timestamp.Stamp.Start()
 	err = generateImageArtifacts(*workers, inDirPath, outDirPath, *releaseVersion, *imageTag, tmpDirPath, config)
 	if err != nil {
 		logger.Log.Panic(err)
 	}
+	timestamp.Stamp.RecordToCSV("generateImageArtifacts", "finishing up")
+	csvparser.OutputCSVLog(csvparser.FilepathsToArray(filepath.Dir(*timestampFile)))
```

Comment:
There is no guarantee that `roast` will be the last tool invoked during a build. If we want to print build info out somehow, it will need to be done by a mechanism in the Makefile rather than inside a go tool.

Reply:
Since we are thinking about moving the call to csvparser.OutputCSVLog() to dashboard.go, we will address this suggestion in the next PR.